### PR TITLE
[GOVCMS-4391]: Add scaffold-tooling to the base image.

### PR DESCRIPTION
**Pull request by steveworley**
- Add the scaffold tooling repository to the base image. This will include the necessary deploy scripts to the images.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,7 +36,8 @@
         "drupal/redis":"1.1.0",
```
Comment (tobybellwood) [inline composer.json]: Adding scaffold-tooling would provide the drupal modules (https://github.com/govCMS/scaffold-tooling/blob/master/composer.json#L4-L21) that we currently provide here - so we'll need to iron out the duplication and control from one spot.

**Comment by simesy** [comment]
This solution is a bandaid to the problem of duplicate logic across different repos.

If you build the Drupal/govcms application into this image, why not just do it via govcms8-scaffold-paas? This will make the scaffold-tooling resources available at /app/vendor/govcms/scaffold-tooling, and reduce duplication.

**Comment by tobybellwood** [comment]
down the track yes, but for now this allows us to use the centralised govcms-tooling repo to manage all our scripts!

**Comment by steveworley** [comment]
We can test this change by changing the version to `dev-feature/govcms-4390-sub-scripts-deploy` it should then include all the scripts in vendor/bin.

For example:

```
$ vendor/bin/govcms-enable-modules
GovCMS Deploy :: Enable modules
 [success] Successfully enabled: redis, fast404, clamav, robotstxt, lagoon_logs, environment_indicator
[success]: Completed successfully
```

**Pull request merged**